### PR TITLE
docs(examples): add pinia examples to docs

### PR DESCRIPTION
**Pull request by valcosmos**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/other/pinia/nuxt.config.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { defineNuxtConfig } from 'nuxt'
+
+// https://v3.nuxtjs.org/api/configuration/nuxt.config
+export default defineNuxtConfig({
+  buildModules: [
```
Comment (vencho-mdp) [inline examples/other/pinia/nuxt.config.ts:5]: AFAIK, `buildModules` is deprecated, so I think `modules` option should be used instead.

Comment (valcosmos) [reply examples/other/pinia/nuxt.config.ts]: Thank you very much, it was my oversight, and I have fixed it.😊

**Comment by danielroe** [comment]
Thanks for your work on this! However, I'd suggest that we focus on making sure that this example is in the pinia docs instead. Might also be worth adding some information in the migration guide about how to migrate a store from Nuxt 2 -> Nuxt 3?

**Comment by pi0** [comment]
Thanks for PR @valcosmos 

I agree with you @danielroe that module examples are probably to be in their own docs but pinia is kinda store replacement. I wouldn't mind having an exception for it :)

**Comment by valcosmos** [comment]
> Thanks for your work on this! However, I'd suggest that we focus on making sure that this example is in the pinia docs instead. Might also be worth adding some information in the migration guide about how to migrate a store from Nuxt 2 -> Nuxt 3?
> 
> WDYT @pi0?

Hello, Thanks for u reply!  Should I close this PR?

**Comment by pi0** [comment]
Let's close it for now. We might add pinia as default store implementation for Nuxt later on. Currently there are no external module integrations in examples dir.

Thanks again for contributing @valcosmos <3

**Pull request closed without merging**